src/pages/workspaces/workspace/analysis/modals/ComputeModal/AzureComputeModal.js

```diff
@@ -342,13 +344,21 @@ export const AzureComputeModalBase = ({
       div({ style: { padding: '1.5rem', overflowY: 'auto', flex: 'auto' } }, [
         renderApplicationConfigurationSection(),
         renderComputeProfileSection(),
-        h(PersistentDiskSection, { persistentDiskExists, computeConfig, updateComputeConfig, setViewMode, cloudPlatform: cloudProviderTypes.AZURE }),
```

Comment:
Again - no compute config being passed down to child components. They don't need that info!